### DISCUSSION TITLE
How to enable with the experimental OpenGL backend in examples ?

### DISCUSSION THREAD

**Discussion by 9prady9** [alandefreitas/matplotplusplus#138] [Q&A] [answered]
I was able to build the OpenGL backend but I am still not clear on how to enable it's use for the actual plots.

Thank you.

**Comment by alandefreitas** [comment, marked as answer]
Hi. This backend is experimental and very incomplete. This discussion might help you understand how things are https://github.com/alandefreitas/matplotplusplus/issues/137
It’s mostly a starting point for contributors. 

We are working on a pre release for a more complete OpenGL/Vulcan/webgl backend.
